eventstream.go

```diff
@@ -45,16 +45,17 @@ const (
 
 // Event defines the structure of event
 type Event struct {
-	ID        string                 `json:"id"`
-	EventName string                 `json:"name"`
-	Namespace string                 `json:"namespace"`
-	ClientID  string                 `json:"clientId"`
-	TraceID   string                 `json:"traceId"`
-	UserID    string                 `json:"userId"`
-	SessionID string                 `json:"sessionId"`
-	Timestamp string                 `json:"timestamp"`
-	Version   int                    `json:"version"`
-	Payload   map[string]interface{} `json:"payload"`
+	ID          string                 `json:"id"`
+	EventName   string                 `json:"name"`
+	Namespace   string                 `json:"namespace"`
+	ClientID    string                 `json:"clientId"`
+	TraceID     string                 `json:"traceId"`
+	SpanContext string                 `json:"spanContext"`
```

Comment:
i think we need to add description to README.md

Reply:
@purbopanambang agree